lighthouserc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "ci": {
+    "collect": {
+      "startServerCommand": "pnpm preview",
```

Comment:
question: doesn't that only work after you built the server? or are you relying on Cypress to build the application?

suggestion: how about we move it into a separate workflow and ensure the application is built? The dependency on Cypress is kind of implicit and flaky